tests/src/common.hh

```diff
@@ -99,6 +117,30 @@ struct ValidCornerFactory {
   }
 };
 
+template <int gridDim>
+auto createUGGridFromCorners(const bool& isRandomlyDistorted, const bool& isFixedDistorted = false) {
```

Comment:
use here also the enum from above

Reply:
You can also pass here the geometry type. This will be handy for later test with triangles or hexahedrons